sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -1987,3 +1989,98 @@ case class GpuStringInstr(str: Expression, substr: Expression)
     }
   }
 }
+
+
+
+class GpuConvMeta(
+  expr: Conv,
+  conf: RapidsConf,
+  parent: Option[RapidsMeta[_,_,_]],
+  rule: DataFromReplacementRule) extends TernaryExprMeta(expr, conf, parent, rule) {
+
+  override def tagExprForGpu(): Unit = {
+    val fromBaseLit = GpuOverrides.extractLit(expr.fromBaseExpr)
+    val toBaseLit = GpuOverrides.extractLit(expr.toBaseExpr)
+    (fromBaseLit, toBaseLit) match {
+      case (Some(Literal(fromBaseVal, IntegerType)), Some(Literal(toBaseVal, IntegerType)))
+        if Set(fromBaseVal, toBaseVal).subsetOf(Set(10, 16)) => ()
+      case _ =>
+        willNotWorkOnGpu(because = "only literal 10 or 16 for from_base and to_base are supported")
+    }
+    if (SQLConf.get.ansiEnabled) {
```

Comment:
ANSI only shows up in 3.4.0+ https://issues.apache.org/jira/browse/SPARK-42427 and if the expression is in ANSI mode or not should come from `expr` not directly from `SQLConf.get.ansiEnabled`.